tests/serializers/test_list_tuple.py

```diff
@@ -160,8 +160,8 @@ class RemovedContains(ImplicitContains):
         ({4.2}, 'Input should be a valid integer, got a number with a fractional part'),
         ({'a'}, 'Input should be a valid integer, unable to parse string as an integer'),
         (ImplicitContains(), 'Input should be a valid set'),
-        (ExplicitContains(), re.compile('.*Invalid Schema:.*Input should be a valid set.*', re.DOTALL)),
-        (RemovedContains(), re.compile('.*Invalid Schema:.*Input should be a valid set.*', re.DOTALL)),
+        (ExplicitContains(), re.compile('.*1 validation error for Schema.*Input should be a valid set.*', re.DOTALL)),
```

Comment:
I think it could be confusing to start the error with the word "validation".

Might be better to use "Invalid Schema:", should be possible by adding a `prefix: &'static str` argument `display()`.